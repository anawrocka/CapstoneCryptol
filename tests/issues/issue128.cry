copy : [inf] -> [inf]
copy ([b] # x) = if b then [True] # copy x else [False] # copy x

property ok = copy ([True] # zero) @ 0
