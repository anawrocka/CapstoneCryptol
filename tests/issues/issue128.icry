:l issue128.cry
:check ok
:prove ok
