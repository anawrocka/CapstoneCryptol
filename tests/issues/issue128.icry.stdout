Loading module Cryptol
Loading module Cryptol
Loading module Main
Using exhaustive testing.
passed 1 tests.
Q.E.D.
Q.E.D.
